### PR TITLE
Fix build

### PR DESCRIPTION
**Pull request by machour**
Included and used our own phpunit version.

### CODE REVIEW

File: tests/ActiveFieldTest.php [hunk 1]
```diff
@@ -55,7 +55,7 @@ public function testRadioList()
         $expectedHtml = <<<HTML
 <div class="form-group field-dynamicmodel-attributename">
 <label class="control-label">Attribute Name</label>
-<input type="hidden" name="DynamicModel[attributeName]" value=""><div id="dynamicmodel-attributename"><div class="radio"><label><input type="radio" name="DynamicModel[attributeName]" value="1"> name1</label></div>
+<input type="hidden" name="DynamicModel[attributeName]" value=""><div id="dynamicmodel-attributename" role="radiogroup"><div class="radio"><label><input type="radio" name="DynamicModel[attributeName]" value="1"> name1</label></div>
```
Comment (machour) [inline tests/ActiveFieldTest.php]: [yiisoft/yii2#16713](https://github.com/yiisoft/yii2/pull/16713) changed the default output for radio lists

**Comment by machour** [comment]
Last build problem seems to be an incompatibility between hhvm & composer..

**Comment by samdark** [comment]
Let's kill HHVM.

**Comment by machour** [comment]
Done, + added PHP 7.3

**Comment by samdark** [comment]
Merged. Thank you very much!

**Pull request merged**